docs/usage/configuration-options.md

```diff
@@ -729,6 +729,10 @@ Renovate does not do a "longest match" algorithm to pick between multiple matchi
 If you have any uncertainty about exactly which hosts a service uses, then it can be more reliable to use `domainName` instead of `hostName` or `baseUrl`.
 e.g. configure `"hostName": "docker.io"` to cover both `index.docker.io` and `auth.docker.io` and any other host that's in use.
 
+### concurrentRequestLimit
+
+Setting this value will limit the number of concurrent outstanding requests for any host it applies to. It should be used only if a datasource is rate limiting Renovate or having problems with the load.
```

Comment:
```suggestion
Usually the default `concurrentRequestLimit` is fine, but you can use `concurrentRequestLimit` to limit the number of concurrent outstanding requests.
You only need to adjust this setting if a datasource is rate limiting Renovate or has problems with the load.
The limit will be set for any host it applies to.
```

How about something like this?

Reply:
@HonkingGoose I like this forumlation, also I'll add example snippet